jupyter-docker/Dockerfile

```diff
@@ -60,8 +66,15 @@ RUN echo "deb $DEBIAN_REPO/debian jessie main\ndeb $DEBIAN_REPO/debian-security
     python-seaborn \
     python-tk \
     python-numpy \
-    libz-dev \
+    libz-dev
 
+# gsutil separately because it need lsb-release and other prereqs installed above
+RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" \
```

Comment:
adds a new layer, which is Technically Bad since we're only on Docker 1.6 and thus we should be trying to minimize the number of layers we have (according to the internet). GAWB-2924 tracks upgrading Docker to something more efficient.